app/assets/stylesheets/comfy/admin/cms/base.sass

```diff
@@ -212,17 +212,21 @@ body#comfy
   // -- Files ------------------------------------------------------------------
   .cms-files-modal
     z-index: 99999
+    .modal-content
+      height: calc(100vh - 60px)
     iframe
       width: 100%
       height: 100%
       border: 0
 
   &.c-comfy-admin-cms-files
     ul.list
-      input.file-path
+      .file-path
         border: 0
         color: $gray-500
         background: inherit
+        white-space: nowrap
+        user-select: all
```

Comment:
never knew this thing even existed. Seems like a chrome only thing though. Firefox needs prefix for it

Reply:
Prefixer are generally done by autoprefixer in the client app. `all` is part of the CSS v4 WD (https://drafts.csswg.org/css-ui-4/#propdef-user-select), so very soon this will be available unprefixed in firefox as well.

Reply:
That makes sense. Let's add autoprefixer to the gem's Gemfile

Reply:
Sounds good

Reply:
Sent in #805